setup.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2017-2022 O2 Czech Republic, a.s.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import sys
+import grpc_tools
+import grpc_tools.protoc as protoc
+from setuptools import setup
+
+def get_version():
+  return "0.10.0" 
+
+def mkdirs(path):
+  if not os.path.exists(path):
+    parent = "/".join(path.split("/")[0:-1])
+    if parent:
+      mkdirs(parent)
+    os.mkdir(path)
+
+def build_ingest_rpc(name):
+  
+  proto_dir = os.path.split(os.path.abspath(sys.argv[0]))[0] + "/direct/rpc/src/main/proto/"
+  protos = list(filter(lambda x: x.endswith(".proto"), os.listdir(proto_dir)))
+  
+  packages = []
+  imports = ""
+ 
+  mkdirs("./target") 
+    
+  protos = list(map(lambda x: proto_dir + "/" + x, protos))
```

Comment:
nit: you can move this one to line 37, it'll be more clear 
```suggestion
  protos = list(map(lambda x: proto_dir + "/" + x, protos))

    packages = []
  imports = ""

  mkdirs("./target") 

```